resources/scripts/components/server/files/FileManagerContainer.tsx

```diff
@@ -91,7 +111,7 @@ export default () => {
             </div>
             {
                 !files ?
-                    <Spinner size={'large'} centered/>
+                    <Spinner size={'large'} centered />
```

Comment:
Remove space

Reply:
Fixed in https://github.com/pterodactyl/panel/pull/3926/commits/9862de5fd97dade3f9783d4d07a451504e830143